lib/ansible/modules/cloud/amazon/aws_kms.py

```diff
@@ -198,11 +204,11 @@ def do_grant(kms, keyarn, role_arn, granttypes, mode='grant', dry_run=True, clea
         if len(changes_needed) and not dry_run:
             policy_json_string = json.dumps(policy)
             kms.put_key_policy(KeyId=keyarn, PolicyName='default', Policy=policy_json_string)
-    except:
-        raise Exception("{}: // {}".format("e", policy_json_string))
+    except Exception as e:
+        raise Exception("{}: // {}".format(e, policy_json_string))
```

Comment:
Ah! I didn't think about that. Would it be better to get rid of the try/except in this function (which would lead to it being caught with the right traceback in main)?